es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,64 @@
+// const { createElement, noop } = require('../lib');
+import createElement from '../lib/create-element';
```

Comment:
Here you should be importing both `createElement` and `noop` from `lib/index.js` as named imports

```JAVASCRIPT
import { createElement, noop } from '../lib';
```